### DISCUSSION TITLE
Publish my Nuxt app in Bluehost

### DISCUSSION THREAD

**Discussion by FerchoCarcho** [nuxt/nuxt#17623] [Questions] [answered]
Today I want to publish my site in Bluehost, it is a Nuxt app. When I generate the production site with "yarn Build", and then accessing the .nuxt directory to copy all the contents to my bluehost html folder I get just plain text and not the Running site as you can see here [tecnoeducacion.online](http://tecnoeducacion.online/)

If you need more information please let me know. Please tell me if there are more configuration steps before "yarn build"

Thank you for your help.

**Comment by FerchoCarcho** [comment, marked as answer]
I had to "yarn Generate" to make it work, but could anybody explain the difference between Yarn build and Yarn generate

**Reply by pi0** [reply]
`yarn build` builds a server. `yarn generate` first builds a temporary server and then prerenders all possible pages with crawler to the `.output/public` or `dist` directory which you can publish to a static hosting.
